e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Customization', () => {
+    let config;
+
+    beforeEach(() => {
+        // Get config
+        cy.apiGetConfig().then((response) => {
+            config = response.body;
+        });
+
+        // # Login as System Admin
+        cy.apiLogin('sysadmin');
+
+        // visit site config customization page
+        cy.visit('/admin_console/site_config/customization');
+    });
```

Comment:
Please ignore my initial comment.  See my comment to other PR that may be applied into this PR as well - https://github.com/mattermost/mattermost-webapp/pull/4330#discussion_r350756386, thanks

Reply:
Thanks, updated, please review.